client/components/thank-you/index.tsx

```diff
@@ -128,17 +148,23 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 
 	return (
 		<ThankYouContainer className={ classNames( 'thank-you__container', containerClassName ) }>
-			<ThankYouHeader className="thank-you__container-header">
+			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
+				<ThankYouFooterContainer>
+					<h1> { thankYouImageFooter } </h1>
```

Comment:
The `h1` should also have the `wp-brand-font` class applied to make sure we use the correct font. You'll notice the designs are using the same font as the section titles in the page.